src/components/DatasetDownloadOptionsForm.js

```diff
@@ -0,0 +1,397 @@
+          <HelpIcon
+            alt="What does it mean to skip quantile normalization for RNA-seq samples?"
+            url="//docs.refine.bio/en/latest/faq.html#what-does-it-mean-to-skip-quantile-normalization-for-rna-seq-samples"
+          />
+        </Checkbox>
+        {!values.quantile_normalize && (
+          <Alert dismissableKey={`skip_quantile_normalization_${dataSetId}`}>
+            Skipping quantile normalization will make your dataset less
+            comparable to other refine.bio data
+          </Alert>
+        )}
+      </div>
+    )
+  );
+};
+
+export const EmailAddressOptions = ({ children }) => {
+  const {
+    touched,
+    errors,
+    values,
+    handleChange,
+    emailAddress,
+    classPrefix,
+    hasToken,
+    submitCount,
+  } = React.useContext(DatasetDownloadOptionsContext);
+  return (
+    <>
+      {touched.email_address && errors.email_address && (
+        <Error>{errors.email_address}</Error>
+      )}
+      <div className={`${classPrefix}__email-input`}>
+        <Field
+          name="email_address"
+          type="email_address"
+          placeholder="jdoe@example.com"
+          className={classnames(
+            `input-text ${classPrefix}__email-input mobile-p`,
+            {
+              'input--error': touched.email_address && errors.email_address,
+            }
+          )}
+        />
+
+        {children}
+      </div>
+      {!hasToken && (
+        <div className={`${classPrefix}__checkbox`}>
+          {submitCount > 0 && errors.acceptTerms && (
+            <Error>{errors.acceptTerms}</Error>
+          )}
+          <Checkbox
+            name="acceptTerms"
+            checked={values.acceptTerms}
+            onChange={handleChange}
+          >
+            I agree to the{' '}
+            <Link href="/terms" as="/terms">
+              <a className="link" target="_blank" rel="noopener noreferrer">
+                Terms of Use
+              </a>
+            </Link>
+          </Checkbox>
+        </div>
+      )}
+      {(emailAddress === '' || values.email_address !== emailAddress) && (
+        <div className={`${classPrefix}__checkbox`}>
+          <Checkbox
+            name="ccdl_email_ok"
+            checked={values.ccdl_email_ok}
+            onChange={handleChange}
+          >
+            I would like to receive occasional updates from the refine.bio team
+          </Checkbox>
+        </div>
+      )}
+    </>
+  );
+};
+
+export const SubmitDatasetOptionsForm = () => {
+  const { actionText, isSubmitting, classPrefix } = React.useContext(
+    DatasetDownloadOptionsContext
+  );
+  return (
+    <div
+      className={classnames(
+        `${classPrefix}__submit`,
+        'flex-button-container',
+        'flex-button-container--left'
+      )}
+    >
+      <Button type="submit" text={actionText} disabled={isSubmitting} />
+    </div>
+  );
+};
+
+/**
+ *
+ */
+const DatasetDownloadOptionsFormComponent = ({
```

Comment:
Yeah this is setting up for removing redux for managing the original dataset. It's a lot of duplication but I'm not going to be including the refactor for that inside of this because it will require a lot more time to untangle that flow.